src/index.ts

```diff
@@ -56,28 +62,40 @@ export interface GetCallOptions extends CallOptions {
   autoCreate?: boolean;
 }
 
-export interface PushConfig {
-  pushEndpoint: string;
-  attibutes?: Map<string, string>;
+export interface Inventory {
```

Comment:
Pretty sure this is a private member of the Publisher, so I don't think we should be exporting this and it should probably live in the Publisher class.